backends/advanced/src/advanced_omi_backend/routers/modules/system_routes.py

```
@@ -128,6 +129,53 @@ async def delete_all_user_memories(current_user: User = Depends(current_active_u
+
+@router.post("/admin/chat/config/validate")
+async def validate_chat_config(
+    request: Request,
+    current_user: User = Depends(current_superuser)
+):
+    """Validate chat configuration YAML. Admin only."""
+    try:
+        yaml_content = await request.body()
+        yaml_str = yaml_content.decode('utf-8')
+        result = await system_controller.validate_chat_config_yaml(yaml_str)
+        return result
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)